.github/workflows/master_default_tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
@heavelock why not '3.10' here (note the single quotes, I had issues earlier today with 3.10 being interpreted as 3.1 (best is to have them all single-quoted)